### DISCUSSION TITLE
how to scale character mesh properly?

### DISCUSSION THREAD

**Discussion by assmikDuDuDuJ** [Sixze/ALS-Refactored#292] [Q&A]
This might be a dumb question, but how do I do this properly? 
What functions/settings should I look at?
If you change the scale of the mesh, the als animation breaks.
What can be done in this case?

**Comment by MagentaPuppet** [comment]
I don't know too much about rigging but you could try re-rigging the character in blender and scaling up the skeleton. This might help: https://youtu.be/XP7Lzu5CNXc
